### ISSUE TITLE
[ Data ] Extraction and Transformation of Data

### ISSUE THREAD

**Issue by porwalshreyaa** [Ayushpanditmoto/Trading-Bot#6]

**Comment by KushalTrivedi19032005** [comment]
I'm a beginner in Machine Learning & Open Source having good theoretical knowledge on MLDL and practically having worked on supervised and unsupervised learning. Do you think I can contribute to this project @porwalshreyaa  ? If yes, I would be happy to learn and contribute!

**Comment by sakshi-sonii** [comment]
Hello, I hope this message finds you well. I am well experienced in python with 5 stars in hackerrank and consistently learning ML. I want to gain hands on experience by contributing to this project through GSSOC'24. Please give me opportunity to contribute to this project. I hold a google data analytics course certificate of 6 months too. I have already completed a project named AI avatar generation in which I have already worked with sound using the librosa library and lipsync GANs. I have a deep knowledge of python,its libraries and frameworks. Please let me contribute to this isuue.

**Comment by litesh1123** [comment]
I am well-versed in data extraction and transformation. i would like to contribute to it.
please assign me this issue

**Comment by OneSouhardyo** [comment]
I have done Exploratory Data Analysis before and am adept at analysis data to find relevant patterns in it. I also am aware of certain commonly used standards used for training DL models to help get better results. I think I will be able to contribute to this issue. I would very much like to contribute to this.

**Comment by aviralgarg05** [comment]
We can use yfinance library and extract real time data, then just convert it into csv for data training. I have worked on this method and is really easy to implement.

**Comment by OneSouhardyo** [comment]
Yes. Thereafter we can use pandas and numpy to make the analyses even faster. We can finally even use certain models to find correlations if they exist.

**Comment by diptarup794** [comment]
Hi, I want to work on this issue for GSSOC'24. I have sufficient knowledge and previous experience and would be able to get the work done . Kindly assign it to me @Ayushpanditmoto

**Comment by aviralgarg05** [comment]
Tf and keras can be used to train the model. I have worked already a lot on a similar project and would look forward to contribute in the same.

**Comment by bhaveshnegi** [comment]
Hi there! I'm really interested in contributing to this project, especially in the area of Machine Learning. I've worked on several machine learning and data science projects, where I've gained experience in handling and processing diverse datasets. Looking forward to potentially collaborating with you all.

**Comment by ubaid2751** [comment]
Hi there! I am very interested in this topic and also I have made models using Machine Learning. I am a beginner but I am sure that I'll be helpful to you. Please let me contribute to this project.

**Comment by bhanushri12** [comment]
Hello @Ayushpanditmoto,
I have had experience with Deep Learning and ML models, waiting to fix the issue.

**Comment by MahimaRamireddy** [comment]
Hi, I want to work on this issue for GSSOC'24. I have sufficient knowledge and previous experience and would be able to get the work done . Kindly assign it to me @Ayushpanditmoto

**Comment by palayushi293** [comment]
Hi ,, I have a good knowledge of ML ,, I want to fix this issue

Waiting for your response
Best Regards

**Comment by Jashwanth05** [comment]
hello , i have great knowledge on ml worked with some projects please assign this work to me

**Comment by utsabsamadder** [comment]
I have good knowledge of Machine Learning and Deep Learning.I could assist you in this project. I would like to join in this project. I have a good grasp on models like YOLOv8 which I recently used to detect Potholes. 

Thank you and have my Regards.

**Comment by RiaKabra** [comment]
I have worked in machine learning projects before and done internships in the same as well. I have a good idea about normalizing data using parallel computing frameworks. Kindly assign me this task , will ensure i come up with a feasible solution for the same

**Comment by shubhamnakum** [comment]
@Ayushpanditmoto I look forward to contributing to this project and ensuring that the data preparation process is thorough and efficient

**Comment by Laxmanpratapsingh** [comment]
i would like to work on this, please assign this to me

**Comment by aryaVishal1706** [comment]
I think this issue is not assign to anybody can i get this issue

**Comment by Shreya2012p** [comment]
Hi there! I'm excited about the possibility of contributing to this project, particularly in the realm of Machine Learning. With hands-on experience from several machine learning and data science projects, I've honed my skills in managing and processing diverse datasets to deliver impactful results. I'm eager to bring this expertise to your team and collaborate on innovative solutions. Looking forward to the opportunity to work together!

**Comment by Shouryabhardwajj** [comment]
I am interested in this issue. Please assign me this

**Comment by Samik123Mit** [comment]
pls, assign me the issue. I will make sure the task is done.

**Comment by iiishita05** [comment]
could you assign me the issue?
